### PR TITLE
Branch defensive session

### PR DESCRIPTION
**Pull request by jwdavis0200**

### CODE REVIEW

**Review by phyliciachristel — APPROVED** [review]
lgtm 10/10

File: src/main/java/seedu/address/model/person/Session.java [hunk 1]
```diff
@@ -54,7 +54,8 @@ public static boolean isValidSession(String test) {
         }
         String day = test.substring(0, 3);
         String hour = test.substring(4);
-        return day.matches(VALIDATION_REGEX_DAY) && hour.matches(VALIDATION_REGEX_TIME) && isValidDay(day);
+        return day.matches(VALIDATION_REGEX_DAY) && hour.matches(VALIDATION_REGEX_TIME) && isValidDay(day)
+                && isValidHour(hour);
```
Comment (phyliciachristel) [inline src/main/java/seedu/address/model/person/Session.java]: wait just kidding maybe this should be time instead of hour cos ure testing minute also

Comment (phyliciachristel) [reply src/main/java/seedu/address/model/person/Session.java]: but ya other than that lgtm!!

Comment (jwdavis0200) [reply src/main/java/seedu/address/model/person/Session.java]: hmm okkiee thank youu

**Pull request merged**